SU-SWS/stanford_neuro_helper
# Imgimprt

Adds a feeds import hook to parse and fetch images from the 'description' of the item response. 

To Test:
1. Check out this branch
2. Run the importer
3. Review node 7168's body for an inline image. 
4. Review file directory for source file.

path alias to node: /news/expert-pilots-process-multiple-visual-cues-more-efficiently-stanford-and-va-scientists-find
